packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.types.ts

```diff
@@ -48,6 +48,13 @@ export interface IDocumentCardProps extends IBaseProps<IDocumentCard> {
    */
   className?: string;
 
+  /**
+   * Aria role assigned to the documentCard (Eg. button, link).
+   * Use this to override the default assignment.
+   * When onClick is provided, default role will be 'button'. When onClickHref is provided, default role will be 'link'.
```

Comment:
Use `@defaultvalue` to document the default :)

Reply:
it actually have 3 different default value based on onClick, onClickHref props. I think it is more clear to just using comment.

Reply:
You can put the entire existing comment behind `@defaultvalue`. That was what I was proposing.. so..

```suggestion
   * @defaultvalue When `onClick` is provided, default role will be 'button'. When `onClickHref` is provided, default role will be 'link'.
```

Reply:
oh, ok. I thought @defaultvalue can only follow by the actually value.